### ISSUE TITLE
proxy isn't necessary

### ISSUE THREAD

**Issue by jbruening** [Ignis34Rus/LoE-Ghost.Server#1]
the login and data locations can be overridden by a json file named `connection_s.json` located next to loe.exe, with the following contents:

``` javascript
{
  "LoginServer": "loedata.legendsofequestria.com/login.php",
  "DataLocation": "http://loedata.legendsofequestria.com/data/",
  "SkipLogin": false,
  "SkipFileUpdate": false,
  "OtherServers": ["localhost=127.0.0.1:14000"]
}
```

obviously, LoginServer and DataLocation are the paths that are used to perform login and download the skills/items/etc.

OtherServers is a string array that will add servers to the server listing page, with the left hand of the equals being what the server is named as, and the right hand being the host and port (separated by the colon)

SkipLogin will cause the client to skip contacting the LoginServer location, and will go straight to the server listing page, which requires the use of OtherServers to have any servers listed. For obvious reasons, the session key will be an empty string, which the server can accept if you're okay with unauthenticated users.

SkipFileUpdate will cause the game to skip checking for updates to the files at DataLocation

any of these properties can be skipped in the file and they'll be what I listed here, except for OtherServers which is by default an empty array

**Comment by GhostInCell** [comment]
Thanks for pointing.

**Comment by GhostInCell** [comment]
Just noticed something:
WebDataLoader always loads data files from _loedata.legendsofequestria.com_ even if DataLocation overridden.

**Comment by antoc** [comment]
I have made some patched dll for the 2016 client of the game : 
The original files are here : [original.zip](https://github.com/Ignis34Rus/LoE-Ghost.Server/files/381260/original.zip)
The patched dll to connect on 127.0.0.7:1034 : [local.zip](https://github.com/Ignis34Rus/LoE-Ghost.Server/files/381261/local.zip)
It should be right now I guess, just extract local.zip, then replace the file in loe_data/Managed
